src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import books from './books/books';
+import categories from './categories/categories';
+
```

Comment:
Please use `createStore` to create a Redux store as it's the requirement and was requested by the previous code reviewer.

Reply:
Thanks for the clarification.